app/graph/mutations/graphql_crud_operations.rb

```diff
@@ -7,7 +7,20 @@ def self.safe_save(obj, attrs, parent_names = [])
       obj.send(method, value) if obj.respond_to?(method)
     end
     obj.disable_es_callbacks = Rails.env.to_s == "test"
-    obj.save_with_version!
+
+    begin
+      obj.save_with_version!
+    rescue RuntimeError => e
+      if e.message.include?("\"code\":#{LapisConstants::ErrorCodes::const_get('DUPLICATED')}") &&
```

Comment:
You could also do a `JSON.parse` on the message and check for the code, but this is fine too.

Reply:
I tried this before, but we ran into an issue with test `GraphqlController12Test#test_should_not_create_feed_invitation` / `rails test test/controllers/graphql_controller_12_test.rb:69`.

```
[11, 20] in /app/app/graph/mutations/graphql_crud_operations.rb
   11:     begin
   12:       obj.save_with_version!
   13:     rescue RuntimeError => e
   14:       require 'byebug'
   15:       byebug
=> 16:       if e.message.include?("\"code\":#{LapisConstants::ErrorCodes::const_get('DUPLICATED')}") &&
   17:       obj.is_a?(ProjectMedia) &&
   18:       obj.set_fact_check.present? &&
   19:       obj.set_original_claim.present?
   20:         existing_pm = ProjectMedia.find(JSON.parse(e.message)['data']['id'])
(byebug) e
#<RuntimeError: No permission to create FeedInvitation>
(byebug) JSON.parse(e)
*** TypeError Exception: no implicit conversion of RuntimeError into String

nil
(byebug) JSON.parse(e.message)
*** JSON::ParserError Exception: unexpected token at 'No permission to create FeedInvitation'

nil
```